solutions/pytorch_tests/Dockerfile

```diff
@@ -1,7 +1,8 @@
 ARG BASE_IMAGE=python:3.9-buster
-ARG PYTORCH_VERSION=v1.10.0
 
 FROM ${BASE_IMAGE} as test
+ARG PYTORCH_VERSION=v1.1.0
```

Comment:
Why is this change needed?

Reply:
I had an issue building the container prior to moving this line. I'll try it again

Reply:
Please use the docker build command in my comment above.

Reply:
@CyanDevs I see, moving to line 4 is correct. Or keep line 2, and adding `ARG PYTORCH_VERSION` to line 4.